packages/common/src/tokens/basicTokens/data/arbitrum.json

```diff
@@ -0,0 +1,202 @@
+{
+  "0xaf88d065e77c8cC2239327C5EDb3A432268e5831": {
```

Comment:
Do we expect to use the address as a key and place it in the toTokenMap, assuming the key represents the symbol? Additionally, how should we migrate the token list from each logic (e.g., `protocolink-logics/src/logics/morphoblue/tokens/data/mainnet.json`) to the current format?